### PR TITLE
Header

### PR DESCRIPTION
**Pull request by thekarinaten**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -5,16 +5,86 @@
...
+  }
+
+body, html, ul, ol, li, p, h1, h2, h3, h4 {
+  margin: 0px;
+  padding: 0px;
+}
+
+div {
+  box-sizing: border-box;
+  }
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
div {
  box-sizing: border-box;
}
```

File: assets/css/style.css [hunk 2]
```diff
@@ -5,16 +5,86 @@
 
 body {
   margin: 0 auto;
+  }
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
}
```

File: assets/css/style.css [hunk 3]
```diff
@@ -5,16 +5,86 @@
...
   max-width: 1600px;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 36px;
+
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/css/style.css [hunk 4]
```diff
@@ -5,16 +5,86 @@
...
+  box-sizing: border-box;
+  }
+
+.wrapper {
   max-width: 1600px;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
```
Comment (katsu223) [inline assets/css/style.css]: sort alphabetically here and below

File: assets/css/style.css [hunk 5]
```diff
@@ -5,16 +5,86 @@
...
+.socials {
+ align-items: center;
+ display: flex;
+ height: 21px;
+ list-style: none;
 }
+
+.intro-container {
+  background-image: url(/assets/images/background-intro.png);
+  background-position: 50% 0px;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  background-position: 50% 0;
```

File: assets/css/style.css [hunk 6]
```diff
@@ -5,16 +5,86 @@
...
+  background-repeat: no-repeat;
+}
+
+.nav-bar {
+  display: flex;
+}
+
+.menu-item {
+  color: white;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  color: #fff;
```

File: assets/css/style.css [hunk 7]
```diff
@@ -5,16 +5,86 @@
...
+  font-size: 21px;
+  text-transform: capitalize;
+  padding: 45px 30px;
+}
+
+.social-image {
+  height: 23px;
+  width: 23px;
+  padding: 8px 8px;
+  padding-bottom: 2px;
```
Comment (katsu223) [inline assets/css/style.css]: Something wierd

File: assets/css/style.css [hunk 8]
```diff
@@ -5,16 +5,86 @@
...
+}
+
+.title {
+  color: #fff;
+  font-weight: 100;
+  font-size: 90px;
+  font-family: Yeseva One;
```
Comment (katsu223) [inline assets/css/style.css]: Add more fallback fonts

File: assets/css/style.css [hunk 9]
```diff
@@ -5,16 +5,86 @@
...
+  max-width: 970px;
+  margin: 0 auto;
+  text-align: center;
+  
+}
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
}

```
Add blank line in the end on the file

File: assets/css/style.css [hunk 10]
```diff
@@ -5,16 +5,86 @@
...
+
+.title {
+  color: #fff;
+  font-weight: 100;
+  font-size: 90px;
+  font-family: Yeseva One;
+  max-width: 970px;
+  margin: 0 auto;
+  text-align: center;
+
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/images/desktop.ini [hunk 11]
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
+Layer 1.png=@Layer 1.png,0
```
Comment (katsu223) [inline assets/images/desktop.ini]: What is it? Delete it from the commit

File: index.html [hunk 12]
```diff
@@ -9,8 +9,31 @@
...
+            <ul class="socials">
+              <li class="social-item"><img class="social-image" src="assets/images/twitter.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/youtube.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/facebook.png"></li>
+            </ul>
+          </nav>
+        </header>
+    <content>
+      <h2 class="title">the right ingredients for the right food</h2>
+    </content>
```
Comment (katsu223) [inline index.html]: Fix indent

File: index.html [hunk 13]
```diff
@@ -9,8 +9,31 @@
...
+            </ul>
+            <ul class="socials">
+              <li class="social-item"><img class="social-image" src="assets/images/twitter.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/youtube.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/facebook.png"></li>
+            </ul>
+          </nav>
+        </header>
+    <content>
+      <h2 class="title">the right ingredients for the right food</h2>
```
Comment (katsu223) [inline index.html]: Too common class name

**Review by katsu223 — CHANGES_REQUESTED** [review]

**Comment by katsu223** [comment]
Gimme the link to the deploy with it commits

File: assets/images/desktop.ini (hunk 11 above)
Comment (thekarinaten) [reply assets/images/desktop.ini]: how?

File: assets/css/style.css (hunk 1 above)
Comment (katsu223) [reply assets/css/style.css]: you didnt fix it

File: assets/css/style.css [hunk 14]
```diff
@@ -5,16 +5,87 @@
...
+  background-position: 50% 0px;
+  background-repeat: no-repeat;
+}
+
+.menu-bar,
+.socials {
+ align-items: center;
+ display: flex;
+ height: 21px;
+ list-style: none;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  align-items: center;
  display: flex;
  height: 21px;
  list-style: none;
```

File: assets/images/desktop.ini (hunk 11 above)
Comment (katsu223) [reply assets/images/desktop.ini]: Try to google first - then ask me

File: index.html [hunk 15]
```diff
@@ -9,8 +9,30 @@
...
+              <li class="menu-item">reservations</li>
+            </ul>
+            <ul class="socials">
+              <li class="social-item"><img class="social-image" src="assets/images/twitter.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/youtube.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/facebook.png"></li>
+            </ul>
+          </nav>
+        </header>
+      <content>
```
Comment (katsu223) [inline index.html]: Delete this tag

File: index.html [hunk 16]
```diff
@@ -9,8 +9,30 @@
...
+        <h2 class="title">the right ingredients for the right food</h2>
```
Comment (katsu223) [inline index.html]:
```suggestion
        <h3 class="title">the right ingredients for the right food</h3>
```

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: index.html [hunk 17]
```diff
@@ -9,8 +9,27 @@
...
+              <li class="menu-item">reviews</li>
+              <li class="menu-item">reservations</li>
+            </ul>
+            <ul class="socials">
+              <li class="social-item"><img class="social-image" src="assets/images/twitter.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/youtube.png"></li>
+              <li class="social-item"><img class="social-image" src="assets/images/facebook.png"></li>
+            </ul>
+          </nav>
+        </header>
```
Comment (katsu223) [inline index.html]:
```suggestion
        <nav class="nav-bar">
          <ul class="menu-bar">
            <li class="menu-item">home</li>
            <li class="menu-item">about</li>
            <li class="menu-item">ingredients</li>
            <li class="menu-item">menu</li>
            <li class="menu-item">reviews</li>
            <li class="menu-item">reservations</li>
          </ul>
          <ul class="socials">
            <li class="social-item"><img class="social-image" src="assets/images/twitter.png"></li>
            <li class="social-item"><img class="social-image" src="assets/images/youtube.png"></li>
            <li class="social-item"><img class="social-image" src="assets/images/facebook.png"></li>
          </ul>
        </nav>
      </header>
```

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css [hunk 18]
```diff
@@ -5,16 +5,88 @@
...
+.menu-bar,
+.socials {
+  align-items: center;
+  display: flex;
+  height: 21px;
+  list-style: none;
+}
+
+.menu-item {
+  color: #ffffff;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  color: #fff;
```

File: assets/css/style.css [hunk 19]
```diff
@@ -5,16 +5,88 @@
...
+}
+
+.nav-bar {
+  align-items: center;
+  display: flex;
+}
+
+.social-image {
+  height: 23px;
+  padding: 8px 8px;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  padding: 8px;
```

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 19 above)
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/css/style.css [hunk 20]
```diff
@@ -5,16 +5,88 @@
...
+  background-image: url(/assets/images/background-intro.png);
+  background-position: 50% 0;
+  background-repeat: no-repeat;
+}
+
+.menu-bar,
+.socials {
+  align-items: center;
+  display: flex;
+  height: 21px;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/css/style.css [hunk 21]
```diff
@@ -5,16 +5,88 @@
...
+
+.intro-container {
+  background-image: url(/assets/images/background-intro.png);
+  background-position: 50% 0;
+  background-repeat: no-repeat;
+}
+
+.menu-bar,
+.socials {
+  align-items: center;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/css/style.css [hunk 22]
```diff
@@ -5,16 +5,88 @@
...
+
+.menu-item {
+  color: #ffffff;
+  font-size: 21px;
+  padding: 45px 30px;
+  text-transform: capitalize;
+}
+
+.nav-bar {
+  align-items: center;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/css/style.css [hunk 23]
```diff
@@ -5,16 +5,88 @@
...
+  display: flex;
+  height: 36px;
+  justify-content: center;
 }
 
 .header-section {
-  padding-left: 175px;
+  align-items: center;
+  display: flex;
+  height: 110px;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

File: assets/css/style.css [hunk 24]
```diff
@@ -5,16 +5,88 @@
...
+div {
+  box-sizing: border-box;
+}
+
+header {
+  align-items: center;
+  display: flex;
+  height: 36px;
+  justify-content: center;
 }
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
```

**Review by katsu223 — CHANGES_REQUESTED** [review]

**Review by katsu223 — APPROVED** [review]

**Pull request merged**